### DISCUSSION TITLE
Blitz with Svelte

### DISCUSSION THREAD

**Discussion by NoelJacob** [blitz-js/blitz#3575] [Ideas]
Svelte's compiler approach fits well with Blitz's magics. Then, the fact that, although used in production, SvelteKit is rapidly being developed, with few braking changes, makes it easier easier integrating any changes to SvelteKit itself. This makes Blitz.js perfect match to put on top of SvelteKit creating the ultimate webdev DX. I also wish SvelteKit itself also took many lovely enhancements of Blitz.js (but then I guess it might get too opinionated). 
Kudos to you guys for making making stuff a breeze .

**Comment by alexander-mart** [comment]
https://github.com/SBHattarj/svelte-blitz
